packages/Chem/src/widgets/scaffold-tree.ts

```diff
@@ -1136,9 +1154,21 @@ export class ScaffoldTreeViewer extends DG.JsViewer {
       this.updateFilters();
       this._bitOpInput!.value = this.bitOperation;
     }
+    else if (p.name === 'size') {
+      const savedTree = JSON.parse(JSON.stringify(ScaffoldTreeViewer.serializeTrees(this.tree)));
+      this.clear();
+      ScaffoldTreeViewer.deserializeTrees(savedTree, this.tree, (molStr: string, rootGroup: TreeViewGroup) => {
+        return this.createGroup(molStr, rootGroup, false);
+      });
+      this.updateSizes();
+      this.updateUI();
+    }
   }
 
   onFrameAttached(dataFrame: DG.DataFrame): void {
+    while (this.root.firstChild) 
```

Comment:
There is a utility method for that, I think it's us.tools.empty, or something like that

Reply:
Yes, I found ui.empty for such case. Fixed too